src/components/commentCount.tsx

```diff
@@ -0,0 +1,66 @@
+// ----- Imports ----- //
+
+import React, { FC } from 'react';
+import { css, SerializedStyles } from '@emotion/core';
+import { textSans } from '@guardian/src-foundations/typography';
+import { border } from '@guardian/src-foundations/palette';
+import { remSpace } from '@guardian/src-foundations';
+
+import { Format } from 'item';
+import { getPillarStyles } from 'pillar';
+
+
+// ----- Component ----- //
+
+interface Props extends Format {
+    count: number;
+    commentable: boolean;
+}
+
+const styles = (colour: string): SerializedStyles => css`
+    ${textSans.medium({ fontWeight: 'bold' })}
+    border: none;
+    background: none;
+    border-left: 1px solid ${border.secondary};
+    padding-top: ${remSpace[2]};
+    color: ${colour};
+`;
+
+const bubbleStyles = (colour: string): SerializedStyles => css`
+    width: 1rem;
+    display: block;
+    margin-left: auto;
+    fill: ${colour};
+`;
+
+const getStyles = ({ pillar }: Format): SerializedStyles => {
+    const colours = getPillarStyles(pillar);
+
+    return styles(colours.kicker);
+}
+
+const getBubbleStyles = ({ pillar }: Format): SerializedStyles => {
+    const colours = getPillarStyles(pillar);
+
+    return bubbleStyles(colours.kicker);
+}
+
+const CommentCount: FC<Props> = ({ count, commentable, ...format }: Props) => {
+    if (!commentable) {
```

Comment:
This was already in master, so don't feel the need to change but we could default to false earlier on `commentable = fields?.commentable ?? false`

Reply:
I'm sorry, I'm not sure I understand what you mean. This is already a `boolean` at this point isn't it, we don't need to null coalesce it?